src/plans/create/steps/virtual-machines/VirtualMachinesStep.tsx

```diff
@@ -45,7 +47,14 @@ const VirtualMachinesStep: FC = () => {
           control={control}
           rules={{ validate }}
           render={({ field }) => (
-            <VirtualMachinesTable isSelectable value={field.value} onChange={field.onChange} />
+            <VirtualMachinesTable
+              isSelectable
+              value={field.value}
+              onChange={(value) => {
+                field.onChange(value);
+                unregister([NetworkMapFieldId.NetworkMap, StorageMapFieldId.StorageMap]);
```

Comment:
what does unregister does?

Reply:
It removes these fields (their key/value pairs) from the form object.